docs/dip-opt.md

```diff
@@ -109,3 +109,236 @@ Following schematic depicts the process(with boundary extrapolation and variable
 anchor point positioning) for two dimensional correlation between a 5x5 image
 and a 3x3 kernel
 ![](./Images/AnchorPointAndBoundaryExtrapolation.png)
+
+### 2. 2D Erosion(erosion_2d)
+
+dip.erosion_2d performs boundary extrapolation for making the size of output image
+equal to the size of input image and then uses coefficient broadcasting and
+strip mining(CBSM) approach for performing erosion.
+
+Boundary extrapolation can be done using
```

Comment:
No need to copy paste this for each and every op in morphological operations category. 

Create a separate section for morphological operations and then explain each operation in respective subsections.